src/components/tipRecords/AuthorAndDate.vue

```diff
@@ -1,51 +1,71 @@
 <template>
-  <div
-    class="author-and-date"
-    @click.stop
```

Comment:
Can we revert this `@click.stop` and use it instead of one below and one in UserCard?